src/20250814_Multi_GHOInkLaunch/GHOInkLaunch.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "GHO Ink Launch"
+author: "Aave Labs"
+discussions: "https://governance.aave.com/t/arfc-launch-gho-on-ink-set-aci-as-emissions-manager-for-rewards/22664"
+snapshot: "https://snapshot.box/#/s:aavedao.eth/proposal/0xf066b8a7b1c0f3d4edff72a047809e3e1f57578d2335fd769e23561a25efb795"
+---
+
+# Simple Summary
+
+This AIP proposes the expansion of GHO, the native asset of the Aave Protocol, to the Ink network utilizing the Chainlink Cross-Chain Interoperability Protocol (CCIP).
+
+The smart contracts have been refined through multiple stages of design, development, testing, and implementation. Likewise, Certora, the DAO service provider, was engaged to conduct code reviews of the implementation.
+
+# Motivation
+
+Building on the successful GHO's presence on other networks, it is now time to expand GHO to other networks. The Ink ecosystem will bring a new set of opportunities, allowing access to a wide array of integrations with other protocols and tools and ultimately enriching GHO's utility potential.
+
+# Specification
+
+This AIP includes a series of actions required to launch GHO on Ink:
+
+1. Configure new Chainlink CCIP lanes between Ink and the chains where GHO is launched with a rate limit of 1.5M GHO capacity and 300 GHO per second rate.
+2. Configure GhoCcipSteward.
+3. List GHO as a borrowable asset on the Aave Pool, with the risk configuration specified in the ARFC. Then, initial liquidity will be provided to the pool as a security measure to mitigate potential vulnerabilities and facilitate a stable launch.
+4. Set ACI multisig ([0xac140648435d03f784879cd789130F22Ef588Fcd](https://explorer.inkonchain.com/address/0xac140648435d03f784879cd789130F22Ef588Fcd)) as Emission Admin for GHO and aGHO rewards, as specified in the ARFC.
```

Comment:
this point is also not in the AIP

Reply:
Removed here https://github.com/bgd-labs/aave-proposals-v3/pull/802/commits/d9deab272c0698a1113071c710f572b76f7d5e8e